server/src/test/scala/de/frosner/broccoli/instances/NomadInstancesSpec.scala

```diff
@@ -64,21 +89,29 @@ class NomadInstancesSpec
       "include resources in instance tasks" in todo
 
       "fail to get instance tasks if the job wasn't found" in prop { (user: Account, id: String) =>
+        val instanceService = mock[InstanceService]
         val client = mock[NomadClient]
+
+        instanceService
```

Comment:
You duplicate the mock for the `InstanceService`. What do you think about moving it into a separate function?

Reply:
Done.